src/video/video_driver.hpp

```diff
@@ -186,6 +186,13 @@ class VideoDriver : public Driver {
 
 	void GameLoopPause();
 
+	/**
+	* Prevents the system from going to sleep.
+	*
+	* @param inhibited If true, sleep will be disabled. If false, sleep will be enabled.
+	*/
```

Comment:
Coding style aligns these `*`s.
```suggestion
	/**
	 * Prevents the system from going to sleep.
	 *
	 * @param inhibited If true, sleep will be disabled. If false, sleep will be enabled.
	 */
```